examples/distributed-ocis/docker-compose.yml

```diff
@@ -0,0 +1,158 @@
+version: '3.7'
```

Comment:
Just add the document start marker `----` as you did in the other YAML file :) and maybe you want to clean up some empty lines at the end of the file. Beside that LGTM